### PR TITLE
Add example of auto-instrumenting Python Flask application using OpenTelemetry

### PR DESCRIPTION
**Pull request by chtushar**
- Adds example to demonstrate flask instrumentation with OTel

### CODE REVIEW

File: python/with-flask/run.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Check if .env file exists
+if [ ! -f .env ]; then
+  echo ".env file not found!"
+  exit 1
+fi
+
+# Read .env file and export variables
+export $(grep -v '^#' .env | xargs)
```
[the remark this answers is not in the document]
Comment (prathamesh-sonpatki) [reply python/with-flask/run.sh:10]: https://github.com/open-telemetry/opentelemetry-python/issues/3189#issuecomment-1769264773

**Review by prathamesh-sonpatki — CHANGES_REQUESTED** [review]

**Comment by prathamesh-sonpatki** [comment]
@chtushar Please change PR title to

Add example of auto-instrumenting Python Flask application using OpenTelemetry

File: python/with-flask/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+```bash
+OTEL_SERVICE_NAME=flask-app
+OTEL_EXPORTER_OTLP_TRACES_ENDPOINT=https://otlp.last9.io/v1/traces
+OTEL_EXPORTER_OTLP_TRACES_HEADERS="Authorization=<OBTAIN_AUTH_HEADER_FROM_LAST9_DASHBOARD>"
+OTEL_TRACES_EXPORTER=console,otlp
+OTEL_METRICS_EXPORTER=none
+```
+
+- Give permission to execute the `run.sh` script:
```
[the remark this answers is not in the document]
Comment (chtushar) [reply python/with-flask/README.md:31]: `run.sh` is required to load the environment variables.

File: python/flask/README.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+```bash
+export OTEL_SERVICE_NAME=flask-app
+export OTEL_EXPORTER_OTLP_ENDPOINT=https://otlp.last9.io
+export OTEL_EXPORTER_OTLP_HEADERS="Authorization=<BASIC_AUTH_HEADER>"
+export OTEL_TRACES_EXPORTER=console,otlp
+export OTEL_METRICS_EXPORTER=otlp
+```
+
+> Note: `BASIC_AUTH_HEADER` should be replaced with the URL encoded value of the
+> basic authorization header.
```
Comment (prathamesh-sonpatki) [inline python/flask/README.md:35]:
```suggestion
> basic authorization header. Read this post to know how [Python Otel SDK](https://last9.io/blog/whitespace-in-otlp-headers-and-opentelemetry-python-sdk/) handles whitespace in headers for more details.
```

File: python/flask/README.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+3. Obtain the OTLP Auth Header from the [Last9 dashboard](https://app.last9.io).
+   The Auth header is required in the next step.
+
+4. Next, run the commands below to set the environment variables.
+
+```bash
+export OTEL_SERVICE_NAME=flask-app
+export OTEL_EXPORTER_OTLP_ENDPOINT=https://otlp.last9.io
+export OTEL_EXPORTER_OTLP_HEADERS="Authorization=<BASIC_AUTH_HEADER>"
+export OTEL_TRACES_EXPORTER=console,otlp
```
Comment (prathamesh-sonpatki) [inline python/flask/README.md:30]:
```suggestion
export OTEL_TRACES_EXPORTER=otlp
```

**Review by prathamesh-sonpatki — APPROVED** [review]

**Pull request merged**